<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventForm</class>
 <widget class="QWidget" name="EventForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pushButton_addBattery,#pushButton_addLamp,#pushButton_taskId{
	background-color: rgb(78, 154, 6);
color:white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1234</width>
        <height>870</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_engineOnOff">
         <property name="title">
          <string>上下电事件</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>kl15:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_kl15">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>power:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_power">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_taskId">
              <property name="text">
               <string>生成taskId</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_taskId"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>当前定位</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="layout_location"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_">
            <property name="title">
             <string>HmiInfo</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="5">
                <widget class="QLineEdit" name="lineEdit_speed"/>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_odoMetro"/>
               </item>
               <item row="0" column="3">
                <widget class="QLineEdit" name="lineEdit_enduranceMile"/>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>enduranceMileage:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>odoMetro:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>speed:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>revSpeed:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_revSpeed"/>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>gearPosition:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QComboBox" name="comboBox_gear">
                 <item>
                  <property name="text">
                   <string>INVALID</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_P</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_R</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_D</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_F</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>GEAR_N</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>蓝牙信息</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_blueToothName"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>pwd:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_blueToothPwd"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>secretKey:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_blueToothSecretKey"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>mac:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_blueToothMac"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_bmsInfo">
         <property name="title">
          <string>电量信息</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_addBattery">
              <property name="text">
               <string>添加电池</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_battery">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_fault">
         <property name="title">
          <string>故障告警信息</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>mcuFault:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_mcuFault"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>bmsFault:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_bmsFault"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_sidestand">
           <property name="title">
            <string>边撑信息</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radioButton_raise">
              <property name="text">
               <string>收起</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_lower">
              <property name="text">
               <string>落下</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>455</width>
                <height>14</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_gear">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>GearPos</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QComboBox" name="comboBox_gear_single">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>INVALID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_P</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_R</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_D</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_F</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GEAR_N</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_seatBucket">
           <property name="title">
            <string>座桶信息</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="radioButton_open">
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_close">
              <property name="text">
               <string>关闭</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>292</width>
                <height>14</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_lamp">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>灯信息</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="pushButton_addLamp">
              <property name="text">
               <string>添加灯</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_lamp"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
